### ISSUE TITLE
This graph call doesn't return CloudPC enrollmentProfileName value

### ISSUE THREAD

**Issue by sandytsang** [microsoftgraph/microsoft-graph-docs-contrib#8816]
I have tried GET https://graph.microsoft.com/beta/deviceManagement/managedDevices?$select=id,deviceName,enrollmentProfileName,model

The result of CloudPC devices enrollmentProfileName property value is null. Why it's null? 
If I use https://graph.microsoft.com/beta/deviceManagement/managedDevices/{id}?$select=id,deviceName,enrollmentProfileName,model , then it shows CloudPC's enrollmentProfileName value.

Can you please have a graph API call that can list all CloudPC enrollmentProfileName value, so that we don't have to numerate through all devices?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1691c888-7955-2567-d53d-3a03eae9d9eb
* Version Independent ID: 6fd35314-8882-a049-fbb9-6037d818d100
* Content: [List managedDevices - Microsoft Graph v1.0](https://learn.microsoft.com/en-us/graph/api/intune-devices-manageddevice-list?view=graph-rest-1.0&tabs=http)
* Content Source: [api-reference/v1.0/api/intune-devices-manageddevice-list.md](https://github.com/microsoftgraph/microsoft-graph-docs-contrib/blob/main/api-reference/v1.0/api/intune-devices-manageddevice-list.md)
* Product: **intune**
* Technology: **microsoft-graph**
* GitHub Login: @jaiprakashmb
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by Jones-Bones** [comment]
This is true also when $filter is used, e.g., `https://graph.microsoft.com/beta/deviceManagement/managedDevices?filter=devicename eq '{Hostname}'` and it affects not only Cloud PCs, any Windows device with this property like Autopilot is affected.
For what it's worth I am using [Invoke-MgGraphRequest](https://learn.microsoft.com/en-us/powershell/module/microsoft.graph.authentication/invoke-mggraphrequest), but I believe it should act just as normal REST API call would.

Found other differences when this API endpoint is called using Filter (or nothing) vs ID

| Property | ID | Filter | 
| ------------- | ------------- | -------------|
| chassisType | returns for example latop | Always returns Unknown |
| easActivationDateTime | Returns actual timestamp | Always returns 0001-01-01 00:00:00 |
| deviceCategoryDisplayName | Empty, but possibly that's expected for my device | Returns Unknown | 
| retireAfterDateTime | 9999-12-31 23:59:59 | 0001-01-01 00:00:00 |
| roleScopeTagIds | Returns {0}| Returns {} |
| ethernetMacAddress | Retrurns MAC address | Empty |
| enrollmentProfileName | Returns expected value | Empty |
| deviceIdentityAttestationDetail | For my device it is unknown | Empty |
| deviceActionResults | Has expected data | Empty |

Additionally hardwareInformation is empty / incomplete for both scenarios. It is revealed when you make call by device ID and also explicitly select it, e.g., `https://graph.microsoft.com/beta/deviceManagement/managedDevices/{id}?$select=hardwareInformation`
Same is true for physicalMemoryInBytes
